examples/python/gRPC/images/gRPC_pb_sendLabeledImage.py

```diff
@@ -53,9 +53,13 @@
     projectCreated = stubMeta.CreateProject(creation)
     projectuuid = projectCreated.uuid
 
-
 theTime = int(time.time())
 
+# A valid camera intrinsics UUID is needed here for succesful storage of Images
+# Run the gRPC_pb_addCameraIntrinsics.py script to add a new camera intrinsics
+# and paste the id below
+camintrinsics_uuid = "c7efb9b3-a952-465a-8038-2d9c9dc39c29"
```

Comment:
please create a new camera intrinsic here and add it to the server. Use this newly generated intrinsic for the image. Users will overlook this comment and wonder why the simple example isn't working